snap-commands/set-config.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+CONFIG_FILE=""
+BASE64_DATA=""
+COMMAND_NAME="deviceupdate-agent.set-config"
+
+function show_usage() {
+
+echo "Usage: $COMMAND_NAME [OPTIONS]
```

Comment:
Instead of $COMMAND_NAME and echo so can avoid having to escape double quote char, you can use [HEREDOC ](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Here-Documents) with `<<` redirection and `$(basename $0)`:
```sh
cat << EOS
Usage: $(basename $0) [OPTIONS]

Description:
    This script ...
    ...
    Only two config file names are allowed: "du-config.json" and "du-diagnostics-config.json".
    ...
EOS
}

Reply:
Fixed.